assets/dns/daemonset.yaml

```diff
@@ -44,14 +44,44 @@ spec:
           timeoutSeconds: 5
           successThreshold: 1
           failureThreshold: 5
+      - name: dns-node-resolver
+        image: "openshift/origin-cli:v4.0"
```

Comment:
This image reference must come from configuration and be represented in `manifest/image-references`.